package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systelab-components-wdio-test",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
As we have added new methods to current interface, I think that it's the second number the one that needs to be increased here. So, I would set new version to 1.2.0.